package.json

```diff
@@ -80,6 +80,9 @@
     "ember-modifier": "^3.2.7 || ^4.1.0",
     "prop-types": "^15.8.1"
   },
+  "peerDependencies": {
+    "@ember/test-helpers": ">= 3.0.0"
```

Comment:
Do we need to allow `^2.9.3` for ember 3?

Reply:
I have corrected version constrains